### DISCUSSION TITLE
middleware/postgresql/user-guide/antiaffinity

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3021] [Announcements]
# middleware/postgresql/user-guide/antiaffinity

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/middleware/postgresql/user-guide/antiaffinity.html

**Comment by Fan-Lin** [comment]
缺少可以复制的yaml代码

**Comment by ModetaNiu** [comment]
Already Fixed in PR #4862
